pkg/cmd/cluster/bind/bind.go

```diff
@@ -102,21 +107,27 @@ func runBind(opts *Options) error {
 		opts.selectedKafka = cfg.Services.Kafka.ClusterID
 	}
 
-	api := connection.API()
-	kafkaInstance, _, error := api.Kafka().GetKafkaById(context.Background(), opts.selectedKafka).Execute()
+	api := apiConnection.API()
+	kafkaInstance, _, err2 := api.Kafka().GetKafkaById(context.Background(), opts.selectedKafka).Execute()
 
-	if error != nil {
-		return error
+	if err2 != nil {
+		return err2
 	}
 
 	if kafkaInstance.Name == nil {
 		return errors.New(localizer.MustLocalizeFromID("cluster.bind.error.emptyResponse"))
 	}
 
-	err = cluster.ExecuteServiceBinding(logger, *kafkaInstance.Name,
-		opts.namespace,
-		opts.appName,
-		opts.forceCreationWithoutAsk)
+	err = cluster.ExecuteServiceBinding(logger, &cluster.ServiceBindingOptions{
+		ServiceName:             *kafkaInstance.Name,
```

Comment:
Safest option to avoid nil-pointer error:

```suggestion
		ServiceName:             kafkaInstance.GetName(),
```